app/views/products/_form.html.erb

```diff
@@ -0,0 +1,51 @@
+<body>
+<main>
+  <h2><%=action_name%> Product</h2>
+  
+  <section>
+  <%= form_with model: @product do |f| %>
+    <div class="form-group">
+      <label>Category</label>
+      <select class="form-control" name="work[category]"><option value="annual">annual</option>
```

Comment:
Just checking...is 'name="work[category]"' correct? I'm guessing this is 'product[category]', but I've only used select once.